draft-ietf-tls-tls13.md

```diff
@@ -2077,14 +2084,14 @@ The "extension_data" field of this extension contains a
 
        enum {
            /* Elliptic Curve Groups (ECDHE) */
-           obsolete_RESERVED(1..22),
-           secp256r1(23), secp384r1(24), secp521r1(25),
-           obsolete_RESERVED(26..28),
-           x25519(29), x448(30),
+           obsolete_RESERVED(0x0001..0x0016),
+           secp256r1(0x0017), secp384r1(0x0018), secp521r1(0x0019),
+           obsolete_RESERVED(0x001A..0x001C),
+           x25519(0x001D), x448(0x001E),
```

Comment:
I think changing this to hex here is counter productive. Likewise for the next block.